archunit/src/main/java/com/tngtech/archunit/core/domain/JavaConstructor.java

```diff
@@ -64,11 +64,22 @@ public boolean isConstructor() {
         return true;
     }
 
+    @Override
+    @PublicAPI(usage = ACCESS)
+    public boolean isMethod() {
+        return false;
+    }
+
     @PublicAPI(usage = ACCESS)
     public Set<JavaConstructorCall> getCallsOfSelf() {
         return getReverseDependencies().getCallsTo(this);
     }
 
+    @PublicAPI(usage = ACCESS)
+    public Set<JavaConstructorCall> getMethodCallsOfSelf() {
```

Comment:
This method is confusing to me because "get method calls...` sounds to me as if it would return `JavaMethodCall`. But instead it's returning `JavaConstructorCall` which is no "method call". I know where you're coming from, but I think we need to find a different naming here :thinking: Maybe it would be something like `getCallsOfSelfByMethods` or `...FromMethods`? :thinking: Or if we only need it in one place we don't add a new API method and simply filter in the one place :man_shrugging: We should ponder carefully if this is a common use case that should go in the API or not...

Reply:
Method was removed as all calls should be analyzed not only calls done by methods.